src/Components/Publishing/Header/Layouts/Components/FeatureFullscreenHeader.tsx

```diff
@@ -14,7 +15,10 @@ export const FeatureFullscreenHeader: React.SFC<FeatureHeaderProps> = props => {
   } = props
   const url = (hero_section && hero_section.url) || ""
   const isVideo = url.includes("mp4")
-  const src = !isVideo && url.length && resize(url, { width: 1600 })
+  const src =
+    !isVideo &&
+    url.length &&
+    resize(url, { width: !sd.IS_MOBILE ? 2000 : 1600 })
```

Comment:
Since we are toggling for mobile we can probably bump the lower number down to something more like a phone size, potentially 600 would work.

Reply:
That makes sense.